server/cli/flags.go

```diff
@@ -105,6 +105,10 @@ var flagUsage = map[string]string{
         ranges. The scan is slowed as necessary to approximately achieve this
         duration.
 `,
+	"scan-max-idle-time": `
```

Comment:
Definitely don't want to expose flags for these. Let's remember to remove them before anyone starts to rely on them.

I saw an interesting presentation at stw about automated testing of config parameters. System currently works in Java but the prof who gave the talk indicated her grad students would probably be willing to port to go.

Did you know that MySQL has over 4,000 pages of documentation discussing how to set over 1,000 config parameters? Insane. Let's never be that and aggressively police the inclusion of any additional external API surface area.